suplemon/main.py

```diff
@@ -182,14 +190,14 @@ def main_loop(self):
 
                 if event:
                     got_input = True
-                    self.on_input(event)
+                    self.on_input(event)  # Up to 30% processing time
```

Comment:
Oops, didn't clean that up yet. I've been doing some performance analysis and finding out where the most time is consumed. The 30% indicates what proportion the main loops execution is spent on `on_input`.